eval.go

```diff
@@ -369,7 +364,11 @@ nextFileMatch:
 		// Prefer docs with several top-scored matches.
 		fileMatch.addScore("repetition-boost", scoreRepetitionFactor*float64(repetitions), opts.DebugScore)
 
-		fileMatch.addScore("atom", float64(atomMatchCount)/float64(totalAtomCount)*scoreFactorAtomMatch, opts.DebugScore)
+		// atom-count boosts files with matches from more than 1 atom. The
+		// maximum boost is scoreFactorAtomMatch.
+		if atomMatchCount > 0 {
```

Comment:
Could there ever be a case where the file matches, but no atoms match (or would this indicate a bad bug)?

Reply:
I don't think that's possible but we don't guarantee it anywhere. The check is probably redundant, but I would prefer to leave it in for now and pend some time understanding the invariants we can rely on.